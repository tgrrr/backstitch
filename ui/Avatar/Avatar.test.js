import React from 'react'
import renderer from 'react-test-renderer'
import { Avatar } from '.'

it('Avatar: default', () => {
    const component = renderer.create(
        <Avatar>
            <>
                Avatar
            </>
        </Avatar>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
