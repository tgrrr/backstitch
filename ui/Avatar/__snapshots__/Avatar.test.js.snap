// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Avatar "
    data-testid="Avatar"
  >
    <div
      className="MuiAvatar-root MuiAvatar-circle MuiAvatar-colorDefault"
    >
      Avatar
    </div>
  </div>
</div>
`;
